R/eml.R

```diff
@@ -29,6 +29,78 @@ pid_to_eml_other_entity <- function(mn, pids) {
   sysmeta_to_eml_other_entity(sysmeta)
 }
 
+
+#' Create an EML code\code{dataTable} object for a given PID. 
+#' This function generates an \code{attributeList} and \code{physical} and constructs a \code{dataTable}.
```

Comment:
Missing blank line between these two lines.

Reply:
D'oh, I put in the wrong newline.

Reply:
Ha, np. Fixed it after merging.